Makefile

```diff
@@ -29,4 +29,4 @@ install: build
 	cp $(OUTPUT) $(GOBIN)
 
 kab: $(GO_SOURCES) VERSION
-	GO111MODULE=on GOOS=linux GOARCH=amd64 go build -o $(OUTPUT) -v
+	GO111MODULE=on go build -o $(OUTPUT) -v
```

Comment:
duffle runs the installer in a linux container, so even if you are running on mac the binary should be for linux.

Reply:
Yes, I checked that in by mistake. (WIBNI git had the facility somehow to ignore a certain change for a while?)